Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -1111,8 +1111,10 @@ def remove(items, scene):
         self.__textitems = []
 
 
-def main(argv=sys.argv):
+def main(argv=None):
```

Comment:
I'd just skip the argument and always give an empty list to `QApplication`. Nobody calls this `main` except for this module itself.